`--use-env` flag fails

## Description and Context
The `--use-env` flag is supposed to allow the use of env variables instead of needing a `hubspot.config.yml` according to the docs: https://developers.hubspot.com/docs/cms/developer-reference/local-development-cli#environment-variables. The docs specifically outline its usage for GitHub Actions

>The HubSpot CLI supports the use of environment variables, this can be especially useful when creating automations like a GitHub Action.

Version: 3.0.9

### Steps to reproduce
1. Create a GitHub Action that uses `--use-env` flag with `hs` command (or just run from a shell script).

Here's an example from my `deploy.yml`

```yml
    - name: Deploy
      env:
        HUBSPOT_PORTAL_ID: ${{ secrets.HUBSPOT_PORTAL_ID }}
        HUBSPOT_PERSONAL_ACCESS_KEY: ${{ secrets.HUBSPOT_PERSONAL_ACCESS_KEY }}
      run: |
        npx hs upload dist cms-theme-dir --use-env
      shell: bash
```

### Expected behavior

Flag successfully removes need for a hubspot yml config file. It worked just the day prior to this:

Actual behavior: it errors out asking for a config file.

```
Run npx hs upload dist cms-theme-dir --use-env
  npx hs upload dist cms-theme-dir --use-env
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    HUBSPOT_PORTAL_ID: ***
    HUBSPOT_PERSONAL_ACCESS_KEY: ***
Error:  A hubspot.config.yml file could not be found
Error:  No config was found
Error: Process completed with exit code 1.
```

@anthmatic 

The flag works for the `upload` command, but not others like `fetch` (which I have in my actions because I need to pull the latest versions of modules).

Here's an example from a basic shell script

### Upload Command Works:

```sh
#!/usr/bin/env bash
set -o allexport
source .env
set +o allexport

printenv | grep "HUBSPOT"

npx hs upload src/index.js cms-theme-dir/index.js --use-env --debug
```

Output: 

```
HUBSPOT_PERSONAL_ACCESS_KEY=***
HUBSPOT_PORTAL_ID=***

CLI version: 3.0.9
node version: v16.13.0
platform: linux

[DEBUG] Loaded config from environment variables for account ***
[DEBUG] Loaded environment variable config
[DEBUG] Updating config for ***
[DEBUG] Attempting to track usage of "upload" command
[DEBUG] Sending usage event to authenticated endpoint
[SUCCESS] Uploaded file from "src/index.js" to "cms-theme-dir/index.js" in the Design Manager of account ***
```

### Fetch Command Fails

```sh
#!/usr/bin/env bash
set -o allexport
source .env
set +o allexport

printenv | grep "HUBSPOT"

npx hs fetch cms-theme-dir/index.js src/index.js --use-env --debug
```


Output:

```
HUBSPOT_PERSONAL_ACCESS_KEY=***
HUBSPOT_PORTAL_ID=***

Debugging info
==============
CLI version: 3.0.9
node version: v16.13.0
platform: linux

[DEBUG] Loaded config from undefined
[ERROR] A hubspot.config.yml file could not be found
[ERROR] No config was found
```

I also cloned the GitHub Action repo, added a `fetch` command and confirmed it does the same thing there.

```yml
name: HubSpot CMS Deploy
author: HubSpot
description: Publish website files to your HubSpot CMS account
inputs:
  src_dir:
    description: Directory in project to be published
    required: true
  dest_dir:
    description: Directory in account which project will be published to
    required: true
  portal_id:
    description: Target HubSpot account (Should be stored in secrets)
    required: true
  personal_access_key:
    description: HubSpot personal access key (Should be stored in secrets)
    required: true
runs:
  using: "composite"
  steps:
    - name: Fetch # added this step
      env:
        HUBSPOT_PORTAL_ID: ${{ inputs.portal_id }}
        HUBSPOT_PERSONAL_ACCESS_KEY: ${{ inputs.personal_access_key }}
      run: |
        npx @hubspot/cli@3.0.9 fetch ${{ inputs.dest_dir }} ${{ inputs.src_dir }} --use-env
      shell: bash

    - name: Deploy
      env:
        HUBSPOT_PORTAL_ID: ${{ inputs.portal_id }}
        HUBSPOT_PERSONAL_ACCESS_KEY: ${{ inputs.personal_access_key }}
      run: |
        npx @hubspot/cli@3.0.9 upload ${{ inputs.src_dir }} ${{ inputs.dest_dir }} --use-env
      shell: bash
```

Result in GH Actions:

```
Run ./
Run npx @hubspot/cli@3.0.9 fetch resources resources --use-env
  npx @hubspot/cli@3.0.9 fetch resources resources --use-env
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    HUBSPOT_PORTAL_ID: ***
    HUBSPOT_PERSONAL_ACCESS_KEY: ***
npx: installed 514 in 24.388s
Error:  A hubspot.config.yml file could not be found
Error:  No config was found
Error: Process completed with exit code 1.
```

--- a/packages/cli/commands/fetch.js
+++ b/packages/cli/commands/fetch.js
@@ -31,7 +31,7 @@ exports.handler = async options => {
   setLogLevel(options);
   logDebugInfo(options);
 
-  loadConfig(configPath);
+  loadConfig(configPath, options);
   checkAndWarnGitInclusion();
 
   if (

